.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,45 @@
+name: "Bug Report"
+description: "Submit a bug report to help us improve"
+title: "Bug Report: "
+labels: [bug,untriaged]
```

Comment:
Should the labels be enclosed in a list(`[]`), or can they just be comma-separated? I see both kinds of labels. Can you make them consistent, please?